examples/glue/run_glue_trainer.py

```diff
@@ -185,41 +185,62 @@ def merge_hparams(hparams: TrainerHparams, override_hparams: GLUETrainerHparams)
 
 def spawn_finetuning_jobs(
     task_to_save_ckpt: Dict[str, bool],
-    ckpt_load_path: str,
+    ckpt_load_paths: List[str],
     ckpt_save_folder: str,
     glue_metrics: GlueState,
     base_yaml_file: str,
     save_locally: bool,
-    parent_ckpt: Optional[str] = None,
+    parent_ckpts: Optional[List[str]] = None,
     load_ignore_keys: Optional[List[str]] = None,
 ) -> None:
-    """Set up CUDA environment and process pool for given finetuning jobs."""
+    """Set up CUDA environment and process pool for given finetuning jobs and wait for them to complete."""
+    if parent_ckpts:
+        assert len(parent_ckpts) == len(ckpt_load_paths), 'Must supply one parent_ckpt per ckpt_load_path'
+    else:
+        parent_ckpts = ckpt_load_paths  # By default, the "parent checkpoint" is logged simply as the checkpoint
+
+    # To reduce noisiness with these tasks, expand the evaluation to multiple fine-tuning seeds per pre-train checkpoint
+    seed_overrides = {
```

Comment:
Wondering if it makes sense to make a field in `finetune_hparams` to store these overrides along with all the other overrides? That way, someone is not bound to spawning all these seeds if they don't need to run all the different seedings.

Reply:
Will sync offline about this!